optimum/exporters/onnx/convert.py

```diff
@@ -274,8 +274,7 @@ def _run_validation(
 
     # Create ONNX Runtime session
     session_options = SessionOptions()
-    # We could well set ORT_DISABLE_ALL here, but it makes CUDA export with O4 of gpt_neo fail
```

Comment:
Was a little confused by comment, does the export of gpt_neo fails with DISABLE?